packages/app/src/cli/services/context.ts

```diff
@@ -251,12 +253,23 @@ export async function ensureDevContext(options: DevContextOptions): Promise<DevC
     })
   }
 
-  showReusedDevValues({
-    selectedApp,
-    selectedStore,
-    cachedInfo,
-    organization,
-  })
+  if (options.customLogsInfoBox) {
+    renderAppLogsConfigInfo({
+      devStore: selectedStore.shopDomain,
+      selectedStores: options.storeFqdns,
+      org: organization.businessName,
+      appName: selectedApp.title,
+      configFile: cachedInfo!.configFile,
+      resetMessage: resetHelpMessage,
+    })
```

Comment:
I agree this doesn't feel ideal, and it feels like we're a bit stuck since we want to `ensureDevContext`, but don't want the post ensure render of the box it does by default.

Could we have `showReusedDevValues` be the default, but be able to provide our own render function when invoking `ensureDevContext` ?